packages/functions/src/common/BookAppointmentHelper.ts

```diff
@@ -10,21 +10,31 @@ import {
   BookingChange,
   FunctionsApi,
   Hospital,
+  MANUAL_DONOR_ID,
 } from "@zm-blood-components/common";
 import { dbAppointmentToBookedAppointmentApiEntry } from "../utils/ApiEntriesConversionUtils";
 import { notifyOnAppointmentBooked } from "../notifications/BookAppointmentNotifier";
-import { BookAppointmentStatus } from "../../../common/src/functions-api";
+import { BookAppointmentStatus } from "@zm-blood-components/common/src/functions-api";
```

Comment:
Use the `FunctionsApi` you already have imported in line 11

Reply:
done